ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -772,3 +772,38 @@ literals:
 - zone_air_occupied_heating_temperature_setpoint
 - mixed_air_relative_humidity_sensor
 - mixed_air_dewpoint_temperature_sensor
+
+- heating_supply_water_temperature_sensor
+- heating_return_water_temperature_sensor
+- heating_thermal_power_sensor
+- heating_thermal_energy_accumulator
+
+#added for UK-LON-X1
+
+- heating_wheel_percentage_sensor
```

Comment:
heating wheel what percentage sensor? 

percent of air flowing through it? speed at which its rotating? percent capacity? needs more specificity.